src/Utilities/Compiler/Options/EditorConfigOptionNames.cs

```diff
@@ -216,6 +216,12 @@ internal static partial class EditorConfigOptionNames
         /// </summary>
         public const string LinqChainMethods = "linq_chain_methods";
 
+        /// <summary>
+        /// Boolean option to configure the assumption that IEnumerable type arguments would be enumerated by method invocation or not.
+        /// It does not affect linq_chain_methods.
+        /// </summary>
```

Comment:
There is usually a comment with a link to the related rule.

```suggestion
        /// Configurable rule: CA1851 (https://docs.microsoft.com/visualstudio/code-quality/ca1851).
        /// </summary>
```

Reply:
The doc has not been created now. I think I can update this after the doc is created?

Reply:
OK I think it would be a temp placeholder, i will do this